code/modules/clothing/spacesuits/captain.dm

```diff
@@ -44,6 +38,7 @@
 	armor = list(melee = 65, bullet = 50, laser = 50, energy = 25, bomb = 50, bio = 100, rad = 50)
 	siemens_coefficient = 0.7
 	heat_conductivity = SPACESUIT_HEAT_CONDUCTIVITY
+	head_type = /obj/item/clothing/head/helmet/space/rig/capspace
 
 /obj/item/clothing/suit/armor/captain/old
```

Comment:
Might want to make sure this thing still works as intended since its previous parent is now dead.

Reply:
I did test it a bunch of times. I had some issues so I had to just keep testing. It SHOULD be fine.

Reply:
So it has the right name, description, armor, etc?